provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1624,6 +1624,8 @@ func generateSchema() schema.PackageSpec {
 						Name:        "ConfigMap",
 						Value:       "CONFIG_MAP",
 						Description: "Only aws-auth ConfigMap will be used for authenticating to the Kubernetes API.",
+						DeprecationMessage: "The aws-auth ConfigMap is deprecated. The recommended method to manage access to Kubernetes APIs is Access Entries with the AuthenticationMode API.\n" +
+							"For more information and instructions how to upgrade, see https://docs.aws.amazon.com/eks/latest/userguide/migrating-access-entries.html.",
```

Comment:
It sounds like we had a Pulumi-specific upgrade document, should we consider linking that instead of the baseline aws.amazon.com migration document? Thanks!

Reply:
I want to link this once we have it published in the docs. This is tracked in the ticket for the how-tos